Tools/yocto/targets.conf

```diff
@@ -0,0 +1,77 @@
+# This configuration file uses the syntax from Python's ConfigParser.
+# The syntax is similar to Windows "INI" files.
+#
+# [section1]
+# key1 = value1
+# key2 = value2
+#
+# Each section entry defines a target machine configuration for the yocto
+# cross-toolchain and image builder
+#
+# This are the entries that each section can have
+#
+# 'repo_manifest_path'  : The path to an XML file that contains the manifest
+#                         with the repository data for the 'repo' tool.
+# 'conf_bblayers_path'  : The path to the bblayers.conf file.
+# 'conf_local_path'     : The path to the local.conf file.
+# 'patch_file_path'     : (Optional) the path to a patch file that will be applied
+#                         over the recipes after downloading them with 'repo'.
+# 'image_basename'      : The name of the Yocto image to be build.
+# 'image_extension'     : The extension of the generated image.
+# 'environment[var]'    : Export var=value on the environment.
+#
+#
+# Notes:
+#   - paths can be relative to this file
+#   - the keys and values are case-sensitive.
+#
+
+# ARM 32-bits targets (ARMv7)
+
+[rpi3-32bits-mesa]
+repo_manifest_path = rpi/manifest.xml
+conf_bblayers_path = rpi/bblayers.conf
+conf_local_path = rpi/local-rpi3-32bits-mesa.conf
+patch_file_path = rpi/fix-nativesdk-image-defs.patch
+image_basename = webkit-dev-ci-tools
+image_extension = wic.bz2
+environment[BUILD_WEBKIT_ARGS] = --no-fatal-warnings --cmakeargs="-DENABLE_WPE_QT_API=OFF -DENABLE_THUNDER=OFF -DENABLE_DOCUMENTATION=OFF -DENABLE_INTROSPECTION=OFF -DWPE_COG_PLATFORMS=drm,headless,wayland"
```

Comment:
meta-webkit includes a recipe for sparke-cdm, so maybe you can keep THUNDER on. You'd get some basic EME support at runtime, including clearkey layout tests coverage support.

Reply:
Great tip. I will do this. I have submitted a [PR to meta-webkit wit the changes required there](https://github.com/Igalia/meta-webkit/pull/444) and I will update this patch with the changes required there as well ASAP